cli/args/mod.rs

```diff
@@ -987,6 +987,10 @@ impl CliOptions {
     }
   }
 
+  pub fn enable_future_features(&self) -> bool {
+    std::env::var("DENO_FUTURE").is_ok()
```

Comment:
maybe we should ensure that the parsed value results in boolean true?

Reply:
we only want to check for presence of the env. eg: this should work `DENO_FUTURE=1`

Reply:
`DENO_FUTURE=false` will also enable future features even though user explicitly disabled it.

Looks like there is inconsistency in the codebase right now:
1. https://github.com/denoland/deno/blob/83d72e5c1c0b983f4b63f3f64f9a1de7600985f4/cli/tools/jupyter/mod.rs#L54-L57
2. https://github.com/denoland/deno/blob/1ad754b4123009e01dbecb3b880e7f0545e46c2f/cli/lsp/language_server.rs#L1148-L1152

Reply:
Additionally reading of env vars should be done only once in `Lazy` at the top level. With this setup we will read env var on each invocation of this function.